app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import re
+
+
+class IncorrectCommand(Exception):
+    """Error if command has incorrect format"""
+
+
+def copy_file(command: str) -> None:
+    command_format = r"cp \w+.txt \w+.txt"
+    if not re.match(command_format, command):
+        raise IncorrectCommand(f"{command} has incorrect format")
+    copied_file, new_file = command.split()[1], command.split()[2]
```

Comment:
no need to write command split for 2 times, just write a command.split() and it will work in right way

Reply:
If there is no need to use first arg, u can just use "_"